func/ui.c

```diff
@@ -74,23 +72,18 @@ int requestAmountOfPeople(){
 
 void printListOfAlternativeRecipes(Recipe alternativeRecipes[]){
 
-    int cellLengths[] = {3, 28, 10};
-    const int numberOfCells = sizeof(cellLengths) / sizeof(cellLengths[0]);
 
     printf("\nHere are some alternative versions:\n");
-    printLine("┌", "┬", "┐", numberOfCells, cellLengths);
 
-    for(int r = 0; r < RECIPES_IN_ALTERNATIVES_LIST; r++){
+    for(int i = 0; i < RECIPES_IN_ALTERNATIVES_LIST; i++){
```

Comment:
```suggestion
    for(int r = 0; r < RECIPES_IN_ALTERNATIVES_LIST; r++){
```
Why?

Reply:
hvorfor r?

Reply:
det er et index for hver _<b>r</b>_ecipe

Reply:
så skriv recepie og ikk r